app/controllers/buyers/service_contracts_controller.rb

```diff
@@ -139,6 +139,6 @@ def service
   end
 
   def service_plan(plan_id = params[:service_contract][:plan_id])
-    @service_plan ||= service.service_plans.find(plan_id)
+    @service_plan ||= service.service_plans.find_by(id: plan_id)
```

Comment:
Isn't this the same?

Reply:
@josemigallas `find` raises an error if it doesn't exist and `find_by` doesn't 😄

Reply:
@josemigallas no. If not exists `find` explodes with an exception, and `find_by` returns `nil` and it will run the validation.

Reply:
OK OK don't jump everybody at the same time 😉

Reply:
hahaha.. I didn't see @Martouta answer 😂 .. sorry